stripe-ui-core/src/main/java/com/stripe/android/uicore/utils/StateFlows.kt

```diff
@@ -20,7 +22,9 @@ import kotlinx.coroutines.flow.map
  * @param flow The [Flow] that should be converted to a [StateFlow]
  * @param produceValue The producer of the [StateFlow's] value
  */
-private class FlowToStateFlow<T>(
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Deprecated("Use helpers such as 'mapAsStateFlow' rather than use this class directly")
```

Comment:
```suggestion
@Deprecated("Use helpers such as 'mapAsStateFlow' rather than use this class directly. This is only public to allow for the inline function usage below.")
```